addons/big-save-button/addon.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Larger \"Save Now\" button",
+  "description": "Extends the size of the \"Save Now\" button in the editor, making it easier to click. By default, this addon introduces no visual changes, only usability improvements.",
+  "tags": ["editor", "editorMenuBar", "recommended"],
+  "versionAdded": "1.35.0",
+  "enabledByDefault": true,
+  "credits": [
+    {
+      "name": "BroJac5246",
+      "link": "https://github.com/BroJac5246/"
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
```

Comment:
It would be nice to also include [`updateUserstylesOnSettingsChange`](https://scratchaddons.com/docs/reference/addon-manifest/#updateuserstylesonsettingschange).

Reply:
> Note: I was unable to get the setting to dynamically enable + disable. It's probably pretty simple, but I wasn't able to figure that out. If someone can help with that then that would be amazing :D

✨ [`updateUserstylesOnSettingsChange`](https://scratchaddons.com/docs/reference/addon-manifest/#updateuserstylesonsettingschange) ✨

Reply:
I tried that but I was having some issues with it... I might try again tomorrow.

Reply:
any details? it should work as expected

Reply:
> any details? it should work as expected

It was acting up for me. But maybe I was doing something wrong and didn't realize it. I'm pretty busy but I should be able to work on this tomorrow or Saturday 🤞

Reply:
I tried again today and it didn't work... but then I tried again and it did. I have no idea what happened. All I know is it did work in the end, so hopefully this is fixed.

Reply:
Changes to addon manifests require an extension reload. Maybe that's why?

Reply:
> Changes to addon manifests require an extension reload. Maybe that's why? Anyway, glad to hear it's working.

I did reload it both times it messed up, so I'm not sure what it was.

Hopefully it doesn't matter now.